app/src/context/Exchange.tsx

```diff
@@ -65,7 +65,7 @@ export const ExchangeProvider: React.FC<{
       const splitName = findNames(name);
       if (splitName?.firstName && splitName.lastName && randomTempPassword) {
         api
-          .gpibSignup({
+          .vendorSignup({
             source: VenderEnum.GPIB,
             firstName: splitName?.firstName,
             lastName: splitName?.lastName,
```

Comment:
If checking for `splitName.lastName` in parent if statement, the lastName field wont be optional for `api.gpibSignup` -- recommend changing `lastName: splitName?.lastName` to `lastName: splitName.lastName`  OR  changing the if statement to look for `splitName?.firstName && splitName?.lastName && randomTempPassword`